packages/plugins/sentry/server/middlewares/sentry.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Programmatic upload middleware. We do not want to export it
```

Comment:
Can you update the comment here?
This is not the upload middleware, nor is it programmatic.

Reply:
Ho yes bad copy paste